clearml_agent/commands/worker.py

```diff
@@ -1548,10 +1548,14 @@ def daemon(self, queues, log_level, foreground=False, docker=False, detached=Fal
                         gpu_indexes=gpu_indexes,
                         gpu_queues=dynamic_gpus,
                     )
-                except Exception:
+                except Exception as e:
                     tb = six.text_type(traceback.format_exc())
                     print("FATAL ERROR:")
                     print(tb)
+
+                    if self._session.config["agent.crash_on_exception"]:
```

Comment:
Hi @nielstenboom,

Better to use a safer access with a default, such as:
`if self._session.config.get("agent.crash_on_exception", false):`